index.html

```diff
@@ -17,158 +17,169 @@
         <a class="logo" href="#">Welcome</a>
         <nav>
             <img id="menuicon" src="./icons/hamburguermenu.png" atl="Menu" />
+            <ul class="menutabs">
+                <a href="#"><li>Portfolio</li></a>
+                <a href="#"><li>About</li></a>
+                <a href="#"><li>Portfolio</li></a>
+            </ul>
```

Comment:
Good job your menu is centered and appears only in desktop screen size 👍 there is one small issue to fix is the font weight  please refer again to figma template to fix it 

**figma header**

![figma header links](https://user-images.githubusercontent.com/59707859/220774572-c3e4a603-5a71-46e3-bf9d-968ce1fcac5f.PNG)
 
**your header**
![student header links](https://user-images.githubusercontent.com/59707859/220774688-eef8db6a-3d8a-42ff-bfc2-b7e7f515be4e.PNG)